python/cugraph/jaccard/wjaccard_wrapper.pyx

```diff
@@ -11,18 +11,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from c_jaccard cimport *
-#from c_graph cimport *
+from c_jaccard cimport * 
+from c_graph cimport * 
 from libc.stdint cimport uintptr_t
 from libc.stdlib cimport calloc, malloc, free
 import cudf
 from libgdf_cffi import libgdf
 from librmm_cffi import librmm as rmm
 import numpy as np
+from numpy.core.numeric import result_type
 
 gdf_to_np_dtypes = {GDF_INT32:np.int32, GDF_INT64:np.int64, GDF_FLOAT32:np.float32, GDF_FLOAT64:np.float64}
 
-def nvJaccard_w(input_graph, weights):
+cpdef jaccard_w(input_graph, weights, first=None, second=None):
```

Comment:
Comments in jaccard.pyx also apply to this file

Reply:
Done.